indigo_app/templates/place/detail.html

```diff
@@ -2,196 +2,220 @@
-                              <div class="progress-bar task-badge-open" style="width: {{ doc.task_stats.open_task_ratio }}%" title="open"></div>
-                              <div class="progress-bar task-badge-pending_review" style="width: {{ doc.task_stats.pending_task_ratio }}%" title="pending review"></div>
-                            </div>
-                          {% endif %}
-                        </div>
-
-                        <div class="col-2">
-                          {% if doc.n_annotations %}
-                            <i class="far fa-comments"></i> {{ doc.n_annotations }}
-                          {% endif %}
-                        </div>
-
-                        <div class="col-2">
-                          <span class="time-ago" data-timestamp="{{ doc.updated_at|date:'c' }}">{{ doc.updated_at|date:'Y-m-d H:i' }}</span>
-                          {% if doc.updated_by_user == request.user %}
-                            you
-                          {% else %}
-                            <div class="text-muted">by {% user_profile doc.updated_by_user %}</div>
-                          {% endif %}
-                        </div>
-                      </div>
-                    {% endfor %}
-                  </div>
-                {% endif %}
-
+          <div class="row">
+            <div class="col-3"> Primary ({{ primary_works_count }}) </div>
+            <div class="col-6">
+              <div class="progress" style="border-radius: 3px; height: 20px;">
+                <div class="progress-bar bg-success" role="progressbar" style="width: {{ primary_works_percentage }}%" aria-valuenow="{{ primary_works_percentage }}" aria-valuemin="0" aria-valuemax="100"></div>
+                <div class="progress-bar" role="progressbar" style="width: {{ subsidiary_works_percentage }}%" aria-valuenow="{{ subsidiary_works_percentage }}" aria-valuemin="0" aria-valuemax="100"></div>
+              </div> 
+            </div>
+            <div class="col-3"> Subsidiary ({{ subsidiary_works_count }}) </div>
+          </div>
+        </div>
+      </div>
+      <div class="card mt-3">
+        <div class="card-body">
+    
+          <div class="row">
+            <div class="col-3 text-center">
+              <div>
+                {% include 'indigo_metrics/_progress_ball.html' with size=50 progress=latest_stat.p_breadth_complete %}
               </div>
+              <h4 class="mb-0 mt-3">{{ latest_stat.p_breadth_complete }}%</h4>
+              <h6>Complete</h6>
+            </div>
+    
+            <div class="col-4 text-center align-self-center">
+              <h4 class="mb-0">{{ latest_stat.n_complete_works }} of {{ latest_stat.n_works }}</h4>
+              <h6>Works complete</h6>
+            </div>
+
+            <div class="col-5 text-center align-self-center">
+              <h6>Works up to date as at</h6>
+              <h4>{% now "Y-m-d" %}</h4>
```

Comment:
Don't forget to pull this from the DB (`place.settings`). It may be null, in which case don't include it at all.